openedx/openedx-platform
# [BB-2967] Remove report path from task output data

In a previous PR (https://github.com/edx/edx-platform/pull/19986) to implement a report status UI, a change was made that added the report path to the task output. This was to simplify getting this data during the task progress, so we could update the UI with it. However, this change makes the task output too large which causes it to fail in the last step in many cases. 

This PR updates the code to use alternative method for getting report download link using the report downloads list. 

**JIRA tickets**: OSPR-4931 

**Discussions**: https://github.com/edx/edx-platform/pull/19986#issuecomment-682377341

**Dependencies**: None

**Sandbox URL**: 
    LMS: https://pr24867.sandbox.opencraft.hosting/
    Studio: https://studio.pr24867.sandbox.opencraft.hosting/

**Merge deadline**: ASAP

**Testing instructions**:

1. Visit the instructor dashboard.
2. Generate a problem responses report
3. Check that browser console for requests to the `instructor_task_status` endpoint
4. Without applying this PR, they should include a `report_path` parameter
5. After applying this PR that parameter is no longer present. 

**Reviewers**

**Settings**
```yaml
EDXAPP_FEATURES:
  ENABLE_COMBINED_LOGIN_REGISTRATION: true
```
